packages/bundler-plugin-core/src/utils.ts

```diff
@@ -344,7 +346,7 @@ export function generateModuleMetadataInjectorCode(metadata: any) {
   // The checks are to support as many environments as possible. (Node.js, Browser, webworkers, etc.)
   // We are merging the metadata objects in case modules are bundled twice with the plugin
   return `{
-  var _sentryModuleMetadataGlobal =
+  const _sentryModuleMetadataGlobal =
```

Comment:
I fixed this too because [block scoping rules for `var`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block#block_scoping_rules_with_var_or_function_declaration_in_non-strict_mode) means that it wasn't working as expected

Reply:
I am all for this but wondering whether we should cut a major in case somebody uses the bundler plugins for apps in an env where const isn't a thing yet.